server/fishtest/schemas.py

```diff
@@ -635,7 +635,7 @@ def flags_must_match(run):
 # about non-validation of runs created with the prior
 # schema.
 
-RUN_VERSION = 13
+RUN_VERSION = 14
```

Comment:
I suppose that his must set to 16 after merging #2295

Reply:
Yep. I will rebase.

Reply:
Rebased and increased version.